docs-src/concepts/what-is-cluster-monitoring.md

```diff
@@ -0,0 +1,32 @@
+---
+id: what-is-cluster-monitoring
```

Comment:
I don't see the need for a separate term here.
This could fit into the "What is a Temporal Cluster" under a "monitoring & observability" section.
I don't think Cluster Monitoring needs to be added to glossary.

Reply:
Yep; updated.